src/app/App.js

```diff
@@ -116,6 +126,44 @@ export default function App() {
             </>
           )}
         />
+        <Route
+          exact
+          path="/feedback"
+          render={() => (
+            <>
+              <Header title={'Feedback'} />
+              <FeedbackResultsPage
+                responses={responses}
+                counter={counter}
+                questions={GallupTwelveQuestions}
+              />
+            </>
+          )}
+        />
+        <Route
+          exact
+          path="/feedback/add"
+          render={props => (
+            <>
+              <Header title={'AddFeedback'} />
+              <FeedbackForm
+                questions={GallupTwelveQuestions}
+                onSubmit={handleFeedbackSubmit}
+                history={props.history}
+              />
+            </>
+          )}
+        />
+        <Route
```

Comment:
as discussed you dont use this route an the complete `BarChart.js` for now. think about merging this part into your master because it implements (a lot) unused code and even files.

Reply:
Thanks, that was a bit messy of me... 😞